library/components/TrendArrow.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="trendArrow">
+        <arrow-up-icon v-if="difference > 0" :size="iconSize" />
+        <minus-icon v-else-if="difference === 0" :size="iconSize" />
+        <arrow-down-icon v-else :size="iconSize" />
+    </div>
+</template>
+
+<script>
+export default {
+  name: "TrendArrow",
+  props: {
+    firstNumber: {
+      type: Number,
+      required: true,
+    },
+    secondNumber: {
+      type: Number,
+      required: true,
+    },
+    iconSize:{
+      type: String,
+      default: '20'
```

Comment:
They should be a number no?
otherwise someone could pass in any string and it would get used

Reply:
Yep! The format used elsewhere was string so I just assumed it was a string (maybe the icons accept input like "small" "medium" and "large"?) but numbers work and make more sense.